src/main/resources/default/assets/javascript/blobreffield.js

```diff
@@ -111,7 +111,7 @@ function initBlobSoftRefField(element, blobKeyField, blobStorageSpace, originalU
 
         const currentPath = element.dataset.path || blobStorageSpacePath;
 
-        selectVFSFile(currentPath, blobStorageSpacePath).then(function (selectedValue) {
+        selectVFSFile(currentPath, blobStorageSpacePath, '').then(function (selectedValue) {
```

Comment:
do we need the same fix in tycho?

Reply:
no, the method is already called correctly everywhere else